api/envoy/config/transformer/aws_lambda/v2/api_gateway_transformer.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package envoy.config.transformer.aws_lambda.v2;
+
+option java_package = "io.envoyproxy.envoy.config.transformer.aws_lambda.v2";
+option java_outer_classname = "ApiGatewayTransformerProto";
+option java_multiple_files = true;
+option go_package = "transformer";
+
+
+message ApiGatewayTransformation {
+} 
```

Comment:
why add an empty proto?

Reply:
As I understand it, this was needed to specify the transformer in typed config ([here](https://github.com/solo-io/gloo/pull/6762#discussion_r924705768) is a comment from the original implementation and [here](https://github.com/solo-io/envoy-gloo/blob/9fcbba0f96213546b91a97bcf99fd36009c31bb4/source/extensions/transformers/aws_lambda/api_gateway_transformer.h#L34-L36) is the function where we consume this, for registering the extension I believe)